lazyflow/stype.py

```diff
@@ -181,7 +191,19 @@ def isConfigured(self):
 
 
     def copy_data(self, dst, src):
-        dst[...] = src[...]
+        # Unfortunately, there appears to be a bug when copying masked arrays
+        # ( https://github.com/numpy/numpy/issues/5558 ).
+        # So, this must be used in the interim.
+        if isinstance(dst, numpy.ma.masked_array):
+            src_val = src[...]
+            dst.data[...] = numpy.ma.getdata(src_val)
+            dst.mask[...] = numpy.ma.getmaskarray(src_val)
+            if isinstance(src_val, numpy.ma.masked_array):
+                dst.fill_value = src_val.fill_value
+        elif isinstance(dst, (list, tuple)) or isinstance(src, (list, tuple)):
+            dst[:] = src[:]
```

Comment:
In general, it is better to use `isinstance(dst, collections.Iterable)` for cases like this rather than explicitly listing all the iterable types you can think of...